host/CMakeLists.txt

```diff
@@ -325,13 +375,18 @@ if (UNIX)
 endif ()
 
 if (CMAKE_C_COMPILER_ID MATCHES GNU)
+  target_compile_options(oehostverify PRIVATE -Wjump-misses-init)
   target_compile_options(oehost PRIVATE -Wjump-misses-init)
 endif ()
 
 # TODO: Remove this hard coded output directory.
 set_property(TARGET oehost PROPERTY
   ARCHIVE_OUTPUT_DIRECTORY ${OE_LIBDIR}/openenclave/host)
 
+# TODO: Remove this hard coded output directory.
```

Comment:
Why did you add it? 😂 I don't believe it's necessary to be bin-placing these things.

Reply:
cuz I copied and pasted a bunch of it 😂